### PR TITLE
Added i5 mac mini

### PR DESCRIPTION
**Pull request by Epro95Z28**
I'm skeptical of the i7's score. I got a similar score when my xcode cache wasn't empty.

### CODE REVIEW

**Comment by beeradmoore** [comment]
Do you mean to the 2018 MacMini or that 2014 MBP? 

For the Mac mini the initial build times were much faster with the i7. The rebuild times were similar because I assume a large chunk of that is a single threaded workload, or at least doesn't get accelerated too much with hyper threading.

File: Readme.md [hunk 1]
```diff
@@ -9,8 +9,9 @@ Xcode 10
 🖥 | Computer Model | CPU | RAM | Fresh Build Time | Incremental Build Time | Xcode Version | Date & Commit Hash | New Build System? |
 -- | -------------- | --- | --- | ---------------- | ---------------------- | ---- | ---- | ----------------- |
 ![](assets/mini.png) | Mac Mini, <br /> Late 2012 | 2.3 GHz i7-3615QM | 8 GB | 1:05 | 0:14 | 10.1 | 2018-11-15 ([commit](https://github.com/artsy/eidolon/commit/c4e042680c8cea809b42eb2b1c827a3ea1d780b4)) | :heavy_check_mark:
-💻 | MacBook Pro <br />Retina, 15", Mid 2014 | 2.8 GHz i7-4980HQ | 16 GB | 0:49 | 0:09 | 10.1 | 2018-11-16 ([commit](https://github.com/artsy/eidolon/commit/c4e042680c8cea809b42eb2b1c827a3ea1d780b4)) | :heavy_check_mark:
+💻 | MacBook Pro <br />Retina, 15", Mid 2014 | 2.8 GHz i7-4980HQ | 16 GB | 0:49 | 0:09 | 10.1 | 2018-11-16 () | :heavy_check_mark:
```
Comment (ashfurrow) [inline Readme.md]: Any reason to remove the commit hash here?

Comment (Epro95Z28) [reply Readme.md]: Nope. Just a mistake

**Comment by Epro95Z28** [comment]
> Do you mean to the 2018 MacMini or that 2014 MBP?
> 
> For the Mac mini the initial build times were much faster with the i7. The rebuild times were similar because I assume a large chunk of that is a single threaded workload, or at least doesn't get accelerated too much with hyper threading.

2018 i7. I find it hard to believe it's twice as fast, but that's just my opinion :)

File: Readme.md (hunk 1 above)
Comment (Epro95Z28) [reply Readme.md]: Revised.

**Comment by beeradmoore** [comment]
Slightly higher frequencies, has hyperthreading enabled so has twice as many logical cores. I would expect it. Some of the earlier tests with Xcode 8 show 50% increase between i7 and i5. Maybe Xcode 10 and the new build process for this project it utilises multiple cores better.

EDIT: There is 2 2018 i7 Mac Minis with the same time. I wonder if your i5 isn't performing fully for some reason. I would have expected it to kill the late 2012 Mac mini 2.3 GHz i7-3615QM.

EDIT2: Although it is interesting that the Geekbench results don't show anywhere near double the improvement (although they are synthetic benchmarks) for the [i7](https://browser.geekbench.com/macs/434) vs the [i5](https://browser.geekbench.com/macs/436).

**Comment by Epro95Z28** [comment]
> Slightly higher frequencies, has hyperthreading enabled so has twice as many logical cores. I would expect it. Some of the earlier tests with Xcode 8 show 50% increase between i7 and i5. Maybe Xcode 10 and the new build process for this project it utilises multiple cores better.
> 
> EDIT: There is 2 2018 i7 Mac Minis with the same time. I wonder if your i5 isn't performing fully for some reason. I would have expected it to kill the late 2012 Mac mini 2.3 GHz i7-3615QM.
> 
> EDIT2: Although it is interesting that the Geekbench results don't show anywhere near double the improvement (although they are synthetic benchmarks) for the [i7](https://browser.geekbench.com/macs/434) vs the [i5](https://browser.geekbench.com/macs/436).

I ran it again and got 0:60. Something doesn't seem right to me.

**Comment by beeradmoore** [comment]
Ill grab the same commit (although it is still the same as my originals) and re-run the test. Can you double check activity monitor to see that there is nothing using up a heap of CPU, memory or disk IO?

**Comment by beeradmoore** [comment]
For reference can you be sure you're testing with this, is the latest commit from the Xcode-10 branch, 
https://github.com/artsy/eidolon/archive/xcode-10-updates.zip

**Comment by beeradmoore** [comment]
@Epro95Z28 https://youtu.be/5QRDbGu00T0
Im getting constant 38s when screen recording, constant 32s when not screen recording.

**Comment by Epro95Z28** [comment]
> For reference can you be sure you're testing with this, is the latest commit from the Xcode-10 branch,
> https://github.com/artsy/eidolon/archive/xcode-10-updates.zip

I may have been running the xcode 9 project. Will report back.

**Comment by Epro95Z28** [comment]
> For reference can you be sure you're testing with this, is the latest commit from the Xcode-10 branch,
> https://github.com/artsy/eidolon/archive/xcode-10-updates.zip

35 seconds... this is the result I was expecting :) 
https://imgur.com/a/1DMjuwH

**Comment by beeradmoore** [comment]
Much more reasonable! And sits a lot more into the synthetic benchmark results of the CPUs when you add the time waiting for the simulator to launch (which is how its meant to be timed AFAIK). I think my build time along was 27ish.

File: Readme.md (hunk 1 above)
Comment (Epro95Z28) [reply Readme.md]: Will update PR tomorrow with my new result (35 seconds)

**Review by ashfurrow — APPROVED** [review]
Okay! So this is good to merge @Epro95Z28?

**Comment by Epro95Z28** [comment]
I need to run it again with the simulator popping up. I didn't realize that was part of the criteria. I will run it again when I get home.

**Comment by Epro95Z28** [comment]
I need to run it again with the simulator popping up. I didn't realize that was part of the criteria. I will run it again when I get home.

**Comment by ashfurrow** [comment]
Okay, thanks!

**Comment by Epro95Z28** [comment]
Okay. 0:42 seems to be my consistent time. I'm pretty content with that!

https://www.youtube.com/watch?v=-gZfsoGkRWw&feature=youtu.be

EDIT: Good to merge whenever

**Comment by ashfurrow** [comment]
Thanks again everyone!

**Pull request merged**